src/lazy/text/raw/sequence.rs

```diff
@@ -21,11 +21,11 @@ use crate::{IonResult, IonType};
 // ===== Lists =====
 
 #[derive(Copy, Clone)]
-pub struct RawTextList<'data, E: TextEncoding<'data>> {
+pub struct RawTextList<'data, E: TextEncoding> {
```

Comment:
🪧 Throughout this file you can see the benefit of removing the lifetime from the `TextEncoding` trait.